service/awsconfig/v5/key/key_test.go

```diff
@@ -123,6 +123,7 @@ func Test_ClusterTags(t *testing.T) {
 	expectedID := "test-cluster"
 	expectedTags := map[string]string{
 		"kubernetes.io/cluster/test-cluster": "owned",
+		"KubernetesCluster":                  "test-cluster",
```

Comment:
`KubernetesCluster` is deprecated but adding it back should be OK. But what about naming the tag `giantswarm.io/cluster`.

That still meets the reporting requirement but it's clear we define the cluster ID. WDYT?

Reply:
I dont really mind, I used the same one because then clients have the Cost Analysis history aligned (same key/value)